src/main/java/sirius/biz/storage/layer3/VirtualFile.java

```diff
@@ -1311,6 +1312,18 @@ private boolean performLoadFromUrl(URI url, FetchFromUrlMode mode) {
                             .hint(ProcessLog.HINT_MESSAGE_KEY, MESSAGE_KEY_LOAD_FROM_URL_FAILED)
                             .hint(ProcessLog.HINT_MESSAGE_COUNT, ProcessLog.MESSAGE_TYPE_COUNT_MEDIUM)
                             .handle();
+        } catch (IllegalArgumentException illegalArgumentException) {
```

Comment:
Not sure if we should "blindly" catch all IllegalArgumentExceptions and just assume that its an error in the location header. But as a quick fix with TODO I'm fine with it.